package.json

```diff
@@ -65,7 +65,7 @@
   "scripts": {
     "start": "webpack-dev-server --history-api-fallback --hot --inline",
     "prestart:docker": "npm install && npm run build",
-    "start-docker": "webpack-dev-server --history-api-fallback --host 0.0.0.0 --port 2992 --hot --inline",
+    "start-docker": "webpack-dev-server --history-api-fallback --host 0.0.0.0 --port 80",
```

Comment:
this is still running webpack dev server.  Seems like it needs to run `npm build`.